ibm_catalog.json

```diff
@@ -93,7 +93,366 @@
                 "description": "The Account Infrastructure Base architecture provides a standardized approach to setting up cloud account settings and related audit logging controls as outlined in the IBM Cloud Framework for Financial Services. The architecture includes configurations for IAM settings and access controls that enforce security best practices. Storing Activity Tracker logs in a Cloud Object Storage bucket ensures that the log data is securely retained and easily accessible for analysis, auditing, and compliance purposes. You can use this architecture as a base for deploying additional services and resources in multiple types of accounts, such as administrative or workload accounts."
               }
             ]
-          }
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key",
+              "type": "password"
+            },
+            {
+              "key": "prefix"
+            },
+            {
+              "key": "skip_iam_account_settings"
+            },
+            {
+              "key": "access_token_expiration"
+            },
+            {
+              "key": "active_session_timeout"
+            },
+            {
+              "key": "allowed_ip_addresses"
+            },
+            {
+              "key": "api_creation",
+              "options": [
+                {
+                  "displayname": "Option 1",
+                  "value": "RESTRICTED"
+                },
+                {
+                  "displayname": "Option 2",
+                  "value": "NOT_RESTRICTED"
+                },
+                {
+                  "displayname": "Option 3",
+                  "value": "NOT_SET"
```

Comment:
`Option 1`, `Option 2`, `Option 3` makes no sense to the end user. How are they supposed to know what these options mean when they see them in the dropdown? 
Suggest this: `Restricted`, `Not restricted`, `Do not set`